resolwe_bio/tools/detect_strandedness.sh

```diff
@@ -63,4 +63,11 @@ if [[ -n "${FIELD}" ]]; then
 fi
 
 RUN="import json; print(json.load(open('results/lib_format_counts.json')).get('expected_format', ''))"
```

Comment:
Change the default value to a non-empty string that is not one of the keys in `strand_codes` to avoid printing `strand_codes: bad array subscript` besides the error message.

Reply:
Thanks! Added `not_detected` instead of `''`.